in_toto/resolvers/github.py

```diff
@@ -0,0 +1,37 @@
+from urllib3.util import Url
+from urllib3 import PoolManager
+import json
+import hashlib
+
+def get_resolvable_url(generic_url):
+    #TODO: get resolvable url for commits, release, etc.
+    resolvable_url = Url(scheme='https', host='api.github.com', path=generic_url)
+    return resolvable_url
+
+def get_hashable_representation(generic_url):
+    http = PoolManager()
+    resolvable_url = get_resolvable_url(generic_url)
+    response = http.request('GET', str(resolvable_url), headers={'User-Agent': 'in-toto Reference Implementation'})
+    response_data = json.loads(response.data)
+
+    representation_object = {}
```

Comment:
We should discuss (perhaps synchronously) the fields recorded here, and how / why they're useful wrt the use cases.

Reply:
This also brings up the question of enabling _some_ user input in deciding what data must be captured, though that's likely a larger, ITE-4 related discussion. The easy way out may be allowing the hashable_representation to have certain optional fields that may be unused.

Reply:
A link to the discussion: https://docs.google.com/document/d/14dZ-KKJcz2Uscygb6n02zcsRHRIGQS6jhWtoHWVqU7M/edit#heading=h.agahzfc87zba